2-factor-authentication/test/my-first-askui-test-suite.test.ts

```diff
@@ -6,42 +6,45 @@ describe('jest with askui', () => {
     await aui.mouseLeftClick().exec();
     await aui.click().text().withText('Sign in').exec();
     await aui
-      .typeIn('<Username>')
+      .typeIn('<usename>',{isSecret: true, secretMask:'**'})
       .textfield()
       .below()
       .text()
       .withText('Username or email address')
       .exec();
-
     await aui
-      .typeIn('<Password>')
+      .pressKey('escape')
+      .exec()
+    await aui
+      .typeIn('<password>', {isSecret: true, secretMask:'**'})
       .textfield()
       .below()
       .text()
       .withText('Password')
       .exec();
-    
+
     await aui.pressKey('tab').exec();
     await aui.pressKey('enter').exec();
 
```

Comment:
Where is the opening-of-the-app-part? 😆 Thought, you wanted to add it.

Reply:
coming soon